neo/test/actor/NeoActorTest.scala

```diff
@@ -99,5 +115,53 @@ class NeoActorTest extends Specification {
         """.stripMargin)
       ok
     }
+
+    "handle HTTP request/response to Berry" in {
+      val url = "localhost:9000"
+      val interval = Interval.parse("2015-11-22T00:00:00.000-08:00/2017-02-14T17:44:56.000-08:00")
+      val jsonRequest = Json.parse(
+        s"""
+           |{
+           |  "dataset"     : "twitter.ds_tweet",
+           |  "keywords"    : ["zika"],
+           |  "timeInterval": ${JsObject(Seq(("start", JsNumber(interval.getStartMillis)), ("end", JsNumber(interval.getEndMillis))))},
+           |  "timeBin"     : "day",
+           |  "geoLevel"    : "state",
+           |  "geoIds"      : [37, 51],
+           |  "mergeResult" : ${JsBoolean(false)}
```

Comment:
Because a `JsBoolean` is required in this field, I put a `JsBoolean` directly. `false` also works in this case. I change it in new commit for simplicity.